packages/atlas/src/components/_crt/CrtHoldersTable/CrtHoldersTable.tsx

```diff
@@ -0,0 +1,105 @@
+import styled from '@emotion/styled'
+import { useMemo } from 'react'
+
+import { useMemberships } from '@/api/hooks/membership'
+import { Avatar } from '@/components/Avatar'
+import { FlexBox } from '@/components/FlexBox'
+import { NumberFormat } from '@/components/NumberFormat'
+import { Table, TableProps } from '@/components/Table'
+import { Text } from '@/components/Text'
+import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
+import { getMemberAvatar } from '@/providers/assets/assets.helpers'
+
+const COLUMNS: TableProps['columns'] = [
+  { Header: 'Member', accessor: 'member' },
+  { Header: 'Total', accessor: 'total' },
+  { Header: 'Vested', accessor: 'vested' },
+]
+
+type CrtHolder = {
+  memberId: string
+  total: number
+  vested: number
```

Comment:
Should there be a `isCreator` here ? To show:

![image](https://github.com/Joystream/atlas/assets/6571453/64937d5a-b8ae-454e-bd1e-d06c23efb7ce)